Server/src/lib/chat/userIdToUserConfigMap.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+interface UserConfig {
+  emoji: string;
+  displayName?: string;
+  id: string;
+}
+
+export const userIdToUserConfigMap = new Map<string, UserConfig>();
```

Comment:
May want to add a comment here stating that while we are remembering users in memory, this is a bad idea as it won't persist between restarts

Reply:
Created an ADO item
https://skype.visualstudio.com/SPOOL/_workitems/edit/2691373